jetson/percep/artag_detector.cpp

```diff
@@ -148,24 +157,23 @@ void TagDetector::updateDetectedTagInfo(rover_msgs::Target *arTags, pair<Tag, Ta
     tags.buffer.push_back(0);
     tags.buffer.push_back(0);
 
-  for (uint i=0; i<2; i++){
-    if(tags.id[i] == -1){//no tag found
-        if(tags.buffer[i] <= 20){//send buffered tag until tag is found
+  for (uint i=0; i<2; i++)
+  {
```

Comment:
nit: Could you fix the placement of this curly brace to abide by our style guide?